packages/product-react/src/components/Select/Select.test.tsx

```diff
@@ -119,16 +119,19 @@ describe('Select', () => {
     it('onFocus', () => {
       component.setProps({ onFocus: fauxCallback });
       component.find('select').simulate('focus');
+      expect(component.find('.ray-select--active').exists()).toBe(true);
     });
 
     it('onBlur', () => {
       component.setProps({ onBlur: fauxCallback });
       component.find('select').simulate('blur');
+      expect(component.find('.ray-select--active').exists()).toBe(false);
     });
 
     it('onChange', () => {
-      component.setProps({ onChange: fauxCallback });
+      component.setProps({ onChange: fauxCallback, value: 4 });
       component.find('select').simulate('change');
+      expect(fauxCallback).toHaveBeenCalled();
```

Comment:
if we're adding a value on L132 shouldn't we be asserting that the callback is called with `4`?